tests/node/rpc/structs/GateStatusTests.js

```diff
@@ -0,0 +1,39 @@
+const SDL = require('../../../config.js').node;
+const GateStatus = SDL.rpc.structs.GateStatus;
+
+const Test = require('./../../../Test.js');
+const Validator = require('./../../../Validator.js');
+const BaseStructTests = require('./BaseStructTests');
+
+describe('GateStatusTests', function () {
+    before(function () {
+        this.create = function () {
+            return new GateStatus()
+                .setLocation(Test.GENERAL_GRID)
+                .setStatus(Test.GENERAL_DOORSTATUSTYPE);
+        };
+
+        this.getExpectedParameters = function (sdlVersion) {
+            return {
+                [GateStatus.KEY_LOCATION]: Test.JSON_GRID,
```

Comment:
@ymalovanyi On line 18 does `Test.JSON_GRID` need to be changed to `Test.GENERAL_GRID`?

Reply:
Same as in above comment.